src/System/Taffybar/DBus/Toggle.hs

```diff
@@ -160,7 +160,7 @@ dbusTogglesStartupHook = do
         readMaybe <$> readFile stateFilepath
       else
         return Nothing
-    MV.modifyMVar_ enabledVar $ const $ return $ fromMaybe M.empty mStartingMap
+    modifyMVar_ enabledVar $ const $ return $ fromMaybe M.empty mStartingMap
```

Comment:
I really prefer importing into a namespace as it was here originally.

Reply:
OK, I have put back qualified imports of MVar.